pom.xml

```diff
@@ -152,6 +152,11 @@
             <version>3.7</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>io.confluent</groupId>
```

Comment:
I believe this is an internal repo. I don't think it will build outside Confluent.

Reply:
Removed this dependency

Reply:
> I believe this is an internal repo. I don't think it will build outside Confluent.

Removed the dependency. Thanks for pointing out